Formula/c/cortexso.rb

```diff
@@ -0,0 +1,34 @@
+require "language/node"
+
+class Cortexso < Formula
+  desc "Drop-in, local AI alternative to the OpenAI stack"
+  homepage "https://jan.ai/cortex"
+  url "https://github.com/janhq/cortex.git",
+      tag:      "v0.4.20",
+      revision: "f5e6b2f48f61c74e8a81fca11b4ea7bfe39580fa"
+  license "Apache-2.0"
+  head "https://github.com/janhq/cortex.git", branch: "dev"
+
+  # Upstream creates releases that use a stable tag (e.g., `v1.2.3`) but are
+  # labeled as "pre-release" on GitHub before the version is released, so it's
+  # necessary to use the `GithubLatest` strategy.
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "node"
+
+  def install
+    cd "cortex-js" do
+      system "npm", "install", *Language::Node.local_npm_install_args
+      system "npm", "run", "build:binary"
+      chmod "+x", "cortex"
+      bin.install "cortex"
+    end
+  end
+
+  test do
+    system "#{bin}/cortex", "-h"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks for the great catch, we have added proper test cases in the latest commit.